### PR TITLE
tasks ids are too similar

### PR DESCRIPTION
**Pull request by mahakporwal02**
## Rationale

[//]: # (Briefly explain the reason behind this change.)
Task ids were too similar hence difficult to differentiate

#304 
## Changes
I have reversed the task ids as suggested and the made the first five characters as short id which makes it more convinient

### CODE REVIEW

**Comment by mahakporwal02** [comment]
@kelson @rgaudin Screenshot from 2020-11-29 22-02-39
Now they are fairly randomised
Screenshot from 2020-11-29 22-04-31
Short ids are the first 5 characters

File: dispatcher/backend/src/utils/scheduling.py [hunk 1]
```diff
@@ -115,6 +116,7 @@ def request_a_schedule(
         "priority": priority,
         "worker": worker,
         "config": config,
+        "_id": ObjectId(str(ObjectId())[::-1])
```
Comment (rgaudin) [inline dispatcher/backend/src/utils/scheduling.py:119]: We need a comment here.

File: dispatcher/frontend-ui/src/constants.js [hunk 2]
```diff
@@ -153,7 +153,7 @@ function trim_command(command, columns=79) {  // trim a string to espaced versio
 }
 
 function short_id(id) {  // short id of tasks (last chars)
-  return id.substr(id.length - 5);
+  return id.substr(0,5);
```
Comment (rgaudin) [inline dispatcher/frontend-ui/src/constants.js:156]: code formatting: space after comma.

**Review by rgaudin — CHANGES_REQUESTED** [review]
Looks good! Please fix the minor formatting stuff.

**Comment by rgaudin** [comment]
@mahakporwal02 once you're done with your changes, ensure your code passes the tests (reports auto-included in the PR) and then re-request a review.

**Comment by mahakporwal02** [comment]
@rgaudin Sorry I force pushed a commit because I was unable to push the last commit due to some mistakes I made online in my personal repository

**Review by rgaudin — APPROVED** [review]
:+1 thanks

**Comment by rgaudin** [comment]
Oh and you should link the issue you fix using the right panel of the PR. That way, it will be closed automatically on merge

**Pull request merged**